commands/command_clean.go

```diff
@@ -77,8 +77,9 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 func init() {
 	RegisterSubcommand(func() *cobra.Command {
 		return &cobra.Command{
-			Use: "clean",
-			Run: cleanCommand,
+			Use:    "clean",
+			Run:    cleanCommand,
+			PreRun: resolveLocalStorage,
```

Comment:
Isn't that all of them except clone? I expect all new commands will need it too, would be easier to remember with a utility function.